### ISSUE TITLE
Are you provide the full training code in the futrue?Want see the diff with other model training details

### ISSUE THREAD

**Issue by FlyEgle** [backseason/SCNet#5]

**Comment by backseason** [comment]
You may refer to the official PyTorch example for details: https://github.com/pytorch/examples/tree/master/imagenet

Based on the official example, for the SCNets reported in our paper, we train for 100 epochs with an initial lr of 0.1, which is divided by 10 after every 30 epochs. 
We also add label smoothing with a ratio of 0.1 as commonly done. 
Except for the above differences, other settings are identical to the PyTorch official example.
